frontend/public/module/auth.js

```diff
@@ -118,6 +118,13 @@ export const authSvc = {
   },
 
   login: () => {
-    window.location = window.SERVER_FLAGS.loginURL;
+    // Ensure that we don't redirect to the current URL in a loop
+    // when using local bridge in development mode without authorization.
+    if (
+      window.location.href !== window.SERVER_FLAGS.loginURL &&
+      window.location.pathname !== window.SERVER_FLAGS.loginURL
```

Comment:
I believe `window.SERVER_FLAGS.loginURL` is always the fully qualified URL, so the second check shouldn't be necessary.

Reply:
I thought this also, but at least on a local dev server it's just the pathname. 🤷‍♂️

Reply:
@spadgett Want live with `window.location.pathname !== window.SERVER_FLAGS.loginURL` or should we keep both for the case where the loginURL is an absolute url instead of local path (what it is currently).